pkg/ccl/streamingccl/streamproducer/replication_stream_test.go

```diff
@@ -229,6 +230,42 @@ func testStreamReplicationStatus(
 	checkStreamStatus(t, hlc.MaxTimestamp, expectedStreamStatus)
 }
 
+func TestSpanConfigReplicationStreamSetup(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	serverArgs := base.TestServerArgs{
+		// This test fails when run from within a test tenant. This is likely
+		// due to the lack of support for tenant streaming, but more
+		// investigation is required. Tracked with #76378.
```

Comment:
Let's not add a TODOTestTenantDisabled here.  Let's use TestIsSpecificToStorageLayerAndNeedsASystemTenant or TestControlsTenantsExplicitly. Both seem true in this case.